templates/layout/base.html

```diff
@@ -41,6 +41,9 @@
 
         {{{head.scripts}}}
 
+
+        {{~inject 'zoomSize' theme_settings.zoom_size}}
+        {{~inject 'productSize' theme_settings.product_size}}
```

Comment:
Could you please explain how moving injects helps to resolve this issue?

Reply:
@jairo-bc can you please add some context as to why do we need to move them here instead on keeping them on product page.

Reply:
Basically, product.js (and product.html, where the variables are injected) is not being loaded on "quick view" click. So I put those variables on the upper level. 
cc @BC-tymurbiedukhin @junedkazi